api/src/main/java/org/pmiops/workbench/db/dao/ParticipantCohortAnnotationDao.java

```diff
@@ -15,4 +17,8 @@ ParticipantCohortAnnotation findByAnnotationIdAndCohortReviewIdAndParticipantId(
             @Param("annotationId") long annotationId,
```

Comment:
Annotation ID is the PK... you don't really need to pass in the cohort review ID and participant ID here?

Reply:
We don't.... i"m just calling it with cohort review ID and participant ID for completeness.